components/base/kinematicbase/kinematics.go

```diff
@@ -80,7 +86,14 @@ func WrapWithDifferentialDriveKinematics(
 	}, nil
 }
 
-func (ddk *differentialDriveKinematics) ModelFrame() referenceframe.Model {
+type differentialDriveKinematics struct {
```

Comment:
[opt] slight preference for struct to be at the top of the file.

Reply:
Personally I prefer structs to be directly above their methods